src/batch/src/task/task_execution.rs

```diff
@@ -254,16 +275,18 @@ impl<C: BatchTaskContext> BatchTaskExecution<C> {
             .await?;
 
         // Spawn task for real execution.
-        tokio::spawn(async move {
+        let t_1 = self.clone();
```

Comment:
`t1` `t2` looks a little bit confusing here. Would you please add more comments?

Reply:
using t_1, t_2 is to just make compiler happy because of the `move` block......